cmake/AddFdbTest.cmake

```diff
@@ -182,67 +189,76 @@ function(create_test_package)
       set(src_dir "${src_dir}/")
       string(SUBSTRING ${src_dir} ${dir_len} -1 dest_dir)
       string(SUBSTRING ${file} ${dir_len} -1 out_file)
-      list(APPEND external_files ${CMAKE_BINARY_DIR}/packages/${out_file})
-      file(COPY ${file} DESTINATION ${CMAKE_BINARY_DIR}/packages/${dest_dir})
+      list(APPEND external_files ${STAGE_OUT_DIR}/${out_file})
+      file(COPY ${file} DESTINATION ${STAGE_OUT_DIR}/${dest_dir})
```

Comment:
`file(COPY ...)` only runs whenever cmake runs. I think this a problem as this means that changes to one of those files won't trigger a package rebuild. Instead this should use `add_custom_command`.

I realize that this is a problem that I initially introduced - so feel free to either fix it or ignore it (if you ignore it I can fix this in a later PR).

Reply:
I can do this.